package/MDAnalysis/core/topologyobjects.py

```diff
@@ -107,9 +107,14 @@ def type(self):
     def is_guessed(self):
         return bool(self._guessed)
 
+    # TODO: which one to use?
```

Comment:
I think the `_cmp_key` one, there will be good tests for this in `core/test_topologyobjects`